### PR TITLE
BooksList Tab

### PR DESCRIPTION
**Pull request by sfernandez11**
# Summary
## Features
* Added a BooksList tab with a list of books with book picture, the title and the author
### Android
image

## Trello Card
https://trello.com/c/39D1W203/5-componentes-ii-listado-de-libros

### CODE REVIEW

File: src/config/api.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import Reactotron from 'reactotron-react-native';
 import { create } from 'apisauce';
 
-const baseURL = 'http://wolox.com';
+const baseURL = ' http://192.168.122.1:3005';
```
Comment (IgnacioAbadie) [inline src/config/api.js]: should be on .env file

File: src/config/api.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ api.addMonitor(Reactotron.apisauce);
 export const apiSetup = dispatch => {
   // eslint-disable-line no-unused-vars, prettier/prettier
   if (baseURL === 'http://wolox.com') {
-    // console.warn('API baseURL has not been properly initialized');
+    console.warn('API baseURL has not been properly initialized');
```
Comment (IgnacioAbadie) [inline src/config/api.js]: remove this if

**Review by IgnacioAbadie — CHANGES_REQUESTED** [review]
Just a little change

File: src/config/api.js (hunk 1 above)
Comment (sfernandez11) [reply src/config/api.js]: Done

File: src/config/api.js (hunk 2 above)
Comment (sfernandez11) [reply src/config/api.js]: Done

**Review by IgnacioAbadie — APPROVED** [review]

**Pull request merged**